README.md

```diff
@@ -2,5 +2,43 @@
 **G**enomic **A**nnotation of **M**etagenomic **S**equences
 
 
-Based on the pipeline-template.
+## Introduction
 
+- Pre-process input reads with cutadapt and report with fastqc
+- (Optionally) Merge overlapping reads with flash
+- Analyse pre-processed reads with centrifuge
+
+## Setting up
+
+1. For running the pipeline you first need to have a working [conda](https://conda.io/docs/index.html) installation.
+Most of the calls require the activation of the relevant conda environment, so 
+you will have to create them **beforehand**. The `requirements.yml` files 
+can be found in the `envs` directory.
+
+2. Some [biopet tools](https://github.com/biopet/tools). Latest versions recommended:
```

Comment:
Both of these are now released in bioconda for your convenience ;-).

Reply:
This will be solved after including the conda env.yml for the testing